### PR TITLE
AP-5427 Add custom properties

### PR DESCRIPTION
**Pull request by dreamind**
Adding the ability to add custom props to the BPMN element. Rename Simulation Parameters to Properties and add two tabs for simulation related properties and more general/custom properties.

### CODE REVIEW

File: Apromore-Core-Components/Apromore-BPMN-Editor/Apromore-BPMN-Modeler-JS/app/translate/translations-ja.js [hunk 1]
```diff
@@ -90,5 +90,7 @@ module.exports = {
   'invalid.endTime {beginTime}':	'無効な終了時間：{beginTime}以降の時間を選択してください。',
 
   'details':	'詳細',
-  'N/A':	'該当なし'
+  'N/A':	'該当なし',
+
+  'properties': 'Properties'
```
Comment (raboczi) [inline Apromore-Core-Components/Apromore-BPMN-Editor/Apromore-BPMN-Modeler-JS/app/translate/translations-ja.js:95]: Do we have a procedure for ensuring that new localizations like this one are eventually translated?

**Review by raboczi — APPROVED** [review]
To my modest command of Javascript, this all seems tidy enough.

**Review by Janite — CHANGES_REQUESTED** [review]
The close panel button is missing from the Properties panel.

File: Apromore-Core-Components/Apromore-BPMN-Editor/Apromore-BPMN-Modeler-JS/app/translate/translations-ja.js (hunk 1 above)
Comment (dreamind) [reply Apromore-Core-Components/Apromore-BPMN-Editor/Apromore-BPMN-Modeler-JS/app/translate/translations-ja.js:95]: We're still doing this manual check through excel master sheet in OneDrive.

**Review by Janite — APPROVED** [review]

**Pull request merged**